frontend/src/queries/useBakerListQuery.ts

```diff
@@ -67,6 +67,11 @@ const BakerQuery = gql<BakerListResponse>`
 								finalizationCommission
 								bakingCommission
 							}
+							paydayCommissionRates {
+								transactionCommission
+								finalizationCommission
+								bakingCommission
```

Comment:
This will be called Block reward commission for the users.

Reply:
We have not changed existing fields and classes in the code, where it isn’t effecting the rendering. Changing all code would have been quite comprehensive. 

New classes and fields do however follow the new terminology.